Pass -Teps instead of -Tps to graphviz dot.
Supposed to work with graphviz version 2.20.0 and later.

2009-08-20 Martin von Gagern

References:
http://bugs.gentoo.org/282150
http://bugzilla.gnome.org/show_bug.cgi?id=592516

Index: doxygen-1.5.8/src/dot.cpp
===================================================================
--- doxygen-1.5.8.orig/src/dot.cpp
+++ doxygen-1.5.8/src/dot.cpp
@@ -1887,7 +1887,7 @@ QCString DotClassGraph::writeGraph(QText
     else if (format==EPS) // run dot to create a .eps image
     {
       DotRunner dotRun(baseName+".dot");
-      dotRun.addJob("ps",baseName+".eps");
+      dotRun.addJob("eps",baseName+".eps");
       if (!dotRun.run())
       {
         QDir::setCurrent(oldDir);
@@ -2221,7 +2221,7 @@ QCString DotInclDepGraph::writeGraph(QTe
     {
       // run dot to create a .eps image
       DotRunner dotRun(baseName+".dot");
-      dotRun.addJob("ps",baseName+".eps");
+      dotRun.addJob("eps",baseName+".eps");
       if (!dotRun.run())
       {
         QDir::setCurrent(oldDir);
@@ -2515,7 +2515,7 @@ QCString DotCallGraph::writeGraph(QTextS
     {
       // run dot to create a .eps image
       DotRunner dotRun(baseName+".dot");
-      dotRun.addJob("ps",baseName+".eps");
+      dotRun.addJob("eps",baseName+".eps");
       if (!dotRun.run())
       {
         QDir::setCurrent(oldDir);
@@ -2657,7 +2657,7 @@ QCString DotDirDeps::writeGraph(QTextStr
     {
       // run dot to create a .eps image
       DotRunner dotRun(baseName+".dot");
-      dotRun.addJob("ps",baseName+".eps");
+      dotRun.addJob("eps",baseName+".eps");
       if (!dotRun.run())
       {
         QDir::setCurrent(oldDir);
@@ -2824,7 +2824,7 @@ void writeDotGraphFromFile(const char *i
   if (format==BITMAP) 
     dotRun.addJob(imgExt,imgName);
   else // format==EPS
-    dotRun.addJob("ps",QCString(outFile)+".eps");
+    dotRun.addJob("eps",QCString(outFile)+".eps");
   if (!dotRun.run())
   {
     QDir::setCurrent(oldDir);
@@ -3185,7 +3185,7 @@ QCString DotGroupCollaboration::writeGra
   else if (format==EPS)
   {
     DotRunner dotRun(baseName+".dot");
-    dotRun.addJob("ps",baseName+".eps");
+    dotRun.addJob("eps",baseName+".eps");
     if (!dotRun.run())
     {
       QDir::setCurrent(oldDir);
